awscli/customizations/s3/results.py

```diff
@@ -268,6 +277,9 @@ def _record_progress_result(self, result, **kwargs):
         self._ongoing_progress[
             self._get_ongoing_dict_key(result)] += bytes_transferred
         self.bytes_transferred += bytes_transferred
+        if result.timestamp > self.start_time:
```

Comment:
@jamesls 

Super unlikely but because we capture the start time in the result recorder and we capture timestamps in the subscriber, there is a chance that if a progress result gets created right after the queued result gets created that the timestamp on the progress result is less than the timestamp of when the result processor actually processes that initial queued result. I can put a comment in the code about this if that makes sense?